### ISSUE TITLE
The Nvidia Driver recommended is no longer available for down load

### ISSUE THREAD

**Issue by ReinoOtoko** [ValveSoftware/Proton#1685]
Nvidia driver you tell us to get is not avialible on Ubuntu 18.04,  they tell me it is missing ppa's  and on Nvidia it is no longer availible for download as they are now pushing their new card  they've rolled back the 396 driver to one that is earlier.. the sudo install does not work!   

## I confirm:
- [ ] that I haven't found another request for this feature.
- [x] that I have checked whether there are updates for my system available that
      contain this feature already.

## References [optional] 
do to Ubuntu  security feature I can't get to where the files are being stored it tells me the Z: directory does not exist but that is where all the errors are going.

**Comment by ReinoOtoko** [comment]
I had more information in here about wine but it was left out
1. I did dl a newer version of Winehq   Steam is running it with out issue.  but my games are now crashing repeatedly.  I'd like to know how to run a check and put the older version back in?

**Comment by kisak-valve** [comment]
Hello @ReinoOtoko, please provide the exact message you are seeing. You may benefit from using ppa-purge to remove the PPA and start over.

**Comment by Xavron** [comment]
I don't know what exact version it is but even 396.45 worked just fine to not have a cow over it =)

Is this even a Proton issue anyway? Seems more of a distro/Nvidia problem.

**Comment by ReinoOtoko** [comment]
First of all Nvidia has removed those drivers from there list of qualified
drivers for Linux  just go look
second  the ppa claims that 2 or 3 of the packages  are either wrong
incomplete or missing
third  the program refuses to download any of them..
I've tried 3 times  and they don't reply to my requests about them at
Ubuntu  18.04

**Comment by Xavron** [comment]
In that case, https://github.com/ValveSoftware/Proton/blob/proton_3.7/PREREQS.md should be changed to not have a specific version listed especially due to the fact that newer drivers become available and simply refer to preferring eg 396.54+ (if at all possible) as some people will follow every letter and digit into a volcano.

Valve isn't going to make a user stand in the corner for having no choice deviating from version listed which let's face it, will absolutely totally happen by everyone at some point.

**Comment by steve7233** [comment]
I am also having problems getting a working driver for steamplay.

**Comment by Kira-PH** [comment]
Can you all just confirm that you _can_ get 396.54? That is the version in PREREQS.md, and is listed in the PPA: https://launchpad.net/~graphics-drivers/+archive/ubuntu/ppa

Also **sudo apt install ubuntu-drivers-common**, then run **ubuntu-drivers autoinstall**

**Comment by ReinoOtoko** [comment]
I'm very sorry I made a big mistake!  when I went to Nvidia to look for my
card I got into the wrong place. because I didn't know that Nvidia had
changed the branding of the card I have to 10 series  so I was looking
under geoforce gtx  it took me to an area of obsolete cards,  as I looked
through the drivers and read the info Nividia said they were stopping
support , my mistake, because I did not know 1070 were there as 10 series
 so all of my post is in error   it should be removed.
